Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,240 @@
+located on the right page, at the right place. This is especially important if
+pages are added, removed, or shuffled around. If the addition is large,
+consider whether the page should be split. Consider also whether new text comes
+with a satisfactory structure. For example, does it fit well with the
+surrounding context, or did the author simply use a "note" box instead of
+trying to integrate the new information to the relevant paragraph?
+
+See also :ref:`the recommendations on documentation structure for contributors
+<docs_structure_recommendations>`.
+
+Specific Items to Look Out For
+------------------------------
+
+Backports Labels
+~~~~~~~~~~~~~~~~
+
+See :ref:`the backport criteria for documentation changes
+<backport_criteria_docs>`. Mark the PR for backports by setting the labels for
+all supported branches to which the changes apply, that is to say, all
+supported branches containing the parent features to which the modified
+sections relate.
+
+CODEOWNERS Updates
+~~~~~~~~~~~~~~~~~~
+
+All documentation sources are assigned to cilium/docs-structure for review by
+default. However, when a contributor creates a new page, consider whether it
+should be covered by another team as well so that this other team can review
+the technical aspects. If this is the case, ask the author to update the
+CODEOWNERS file.
+
+Beta Disclaimer
+~~~~~~~~~~~~~~~
+
+When a feature is advertised as Beta in the PR, make sure that the author
+clearly indicates it as such in the documentation, both by mentioning "(Beta)"
+in the title of the section for the feature and by including the dedicated
+banner. Users can include this banner by using the ``include`` directive for
+the ``Documentation/beta.rst`` fragment.
+
+Upgrade Notes
+~~~~~~~~~~~~~
+
+When the PR introduces new user-facing options, metrics, or behavior, in such a
+way that it affects upgrades or downgrades, ensure that the author summarizes
+the changes with a note in ``Documentation/operations/upgrade.rst``.
+
+Completeness
+~~~~~~~~~~~~
+
+Make sure that new or updated content is complete, with no TODOs.
+
+Auto-Generated Reference Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are some reference documents (command reference, Helm reference) that
+should be automatically updated when parts of the Cilium repository change. You
+should ensure that everything is up-to-date, although the CI does in fact
+handle the verification for you. But you may have to accompany contributors and
+help them figure out the command they need to run to perform the update.
+
+Spell Checker Exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Documentation checks include running a spell checker. This spell checker
+uses a file containing a list of exceptions to ignore, to make it possible to
+use technical terms that are not in its dictionary. Team cilium/docs-structure
+is the owner for this file. Usually, there is not much feedback to provide on
```

Comment:
```suggestion
owns this file and its associated script, ``Documentation/update-spelling_wordlist.sh``. Usually, there is not much feedback to provide on
```

Reply:
True, but I'm not sure what this info brings in the context of the instructions for reviewing updates to the spelling exceptions :thinking:. I'd rather have it mentioned as part of the docs build framework, when we describe it?

Reply:
That's fair; the script doesn't need a mention here.